addons/website/views/snippets/s_adventure.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_adventure" name="Adventure">
+    <section class="s_adventure pt200 pb200 oe_img_bg" style="background-image: url('/web/image/website.s_adventure_default_image'); background-position: 50% 0%;">
+        <div class="o_we_bg_filter bg-white-50"/>
+        <div class="container">
+            <h1 style="text-align: center;">
+                <span class="display-5 text-o-color-1">Embark on your</span><br/>
```

Comment:
I think that's better without text primary
| |
|--------|
| ![image](https://github.com/user-attachments/assets/33a589b9-6f33-47f7-9e5b-bd1c755c7d1d) |  

```suggestion
                <span class="display-5">Embark on your</span><br/>
```